src/Common/src/System/Dynamic/Utils/ExpressionUtils.cs

```diff
@@ -141,35 +141,35 @@ public static void ValidateArgumentCount(MethodBase method, ExpressionType nodeK
             }
         }
 
-        public static Expression ValidateOneArgument(MethodBase method, ExpressionType nodeKind, Expression arg, ParameterInfo pi)
+        public static Expression ValidateOneArgument(MethodBase method, ExpressionType nodeKind, Expression arguments, ParameterInfo pi)
```

Comment:
In this assembly it can come from `Expression.Goto()`, `Expression.Invoke()`, `Expression.Call()` or `Expression.New()`, with `arguments` being the correct name each time. It's also called in Dynamic.Runtime, again with the appropriately named arguments.